### PR TITLE
chore(core/strapi): build strapi core with pack-up

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

- Build strapi/strapi with pack-up
- Remove the use of getLocaleScript which hasn't be necessary for a long time and made the build too complexe
- Remove unnecessary export from index

### Why is it needed?

To prepare v5

### How to test it?

CI + experimental release and QA on commands

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: .strapi-updater.json
Comment (joshuaellis) [inline .strapi-updater.json:1]: How come this has suddenly been checked? 👀

**Review by joshuaellis — DISMISSED** [review]
One small question, but not a blocker!

File: .strapi-updater.json
Comment (alexandrebodin) [reply .strapi-updater.json:1]: I did not actually see it I'll remove it and add to gitignore

**Review by joshuaellis — DISMISSED** [review]

**Review by joshuaellis — APPROVED** [review]

**Review by innerdvations — APPROVED** [review]
LGTM!

**Pull request merged**